### DISCUSSION TITLE
disconnect command

### DISCUSSION THREAD

**Discussion by bilalraj3232** [eritislami/evobot#970] [Ideas]
make disconnect leave command for bot

**Comment by Kilo702** [comment]
make a leave.js in the commands list
copy paste this code into that folder:
const i18n = require("../util/i18n");

module.exports = {
  name: "leave",
  cooldown: 3,
  aliases: [,"disconnect","INSERT_OWN_ALIASES"],
  description: i18n.__("Disconnects the Bot"),
  execute(message) {
    const { channel } = message.member.voice;
    channel.leave();
    message
    .channel
    .send(i18n.__("Ok I'll leave :("))
    .catch(console.error);
  }
};

//this should do the trick
